cmd/kubeadm/app/cmd/reset.go

```diff
@@ -234,10 +228,8 @@ func newCmdReset(in io.Reader, out io.Writer, resetOptions *resetOptions) *cobra
 				return err
 			}
 
-			// output help text instructing user how to remove cni folders
-			fmt.Print(cniCleanupInstructions)
 			// Output help text instructing user how to remove iptables rules
```

Comment:
The comment needs to be updated as well.

Reply:
good catch, i removed the comment.